index.js

```diff
@@ -30,32 +128,49 @@ function tokenIntrospect(opts = {}) {
     }
   }
 
-  return function introspect(token, tokenTypeHint) {
-    const data = { token };
-    if (tokenTypeHint) {
-      data.token_type_hint = tokenTypeHint;
+  const remoteIntrospectionInfo = Object.assign({}, options, { proxy });
```

Comment:
Use spread operator, { ...options, proxy}

Reply:
Not supported in Babel without a [separate plugin](https://babeljs.io/docs/plugins/transform-object-rest-spread/) since the [Object spread operator is still a proposal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator#Spread_in_object_literals).